index.html

```diff
@@ -1,68 +1,286 @@
+        />
+        <div class="project-card-iterms flex column">
+          <h2 class="card-title">Tonic</h2>
+          <div class="card-lists">
+            <p class="canopy">Canopy</p>
+            <div class="round"></div>
+            <p>Backend Dev</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="card-note">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="card-tag">
+            <ul class="card-skills">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+
+      <!-----cards 4--------->
+
+      <div class="project-card flex column">
+        <img
+          src="images/Snapshoot Portfolio (3).svg"
+          alt="project card image"
+        />
+        <div class="project-card-iterms flex column">
+          <h2 class="card-title">Multi-Post <br />Stories</h2>
+          <div class="card-lists">
+            <p class="canopy">Canopy</p>
+            <div class="round"></div>
+            <p>Backend Dev</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="card-note">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="card-tag">
+            <ul class="card-skills">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!----------About Section------------->
+    <section class="about">
+      <div class="about-me flex column">
+        <div class="is-me flex column">
+          <h2 class="about-text">About <br />Myself</h2>
+          <p class="self">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+
+          <div class="intro-items">
+            <a href="#">LET'S CONNECT</a>
+            <div class="intro-connect">
+              <ul class="list-items skill-items">
+                <li>
+                  <a href="#"> <img src="images/github.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img src="images/Linkedin icon.svg" alt="" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/angellist.png" alt="" /> </a>
```

Comment:
[OPTIONAL] Kindly make sure you add some text to  ALT attribute. 
ALT text refers to invisible description of images which are read aloud to blind users on a screen reader. Adding ALT text allows authors to include images, but still provide the content in an alternative text based format